docs/cli-tools.md

```diff
@@ -0,0 +1,67 @@
+
+# Command line tools for Openshidft and Kubernetes
+
+This tools allow users to run Kubernetes and Openshift locally, run commands against Kubernetes clusters, and debug workloads running localy or on remote clusters.
```

Comment:
`Openshift` or `OpenShift` :) ?

Reply:
hmm :-)
https://www.redhat.com/en/technologies/cloud-computing/openshift/try-it
looks like `OpenShift`